src/libtorch.cc

```diff
@@ -225,6 +225,29 @@ ModelState::AutoCompleteConfig()
        Name() + "': not supported for pytorch backend")
           .c_str());
 
+  // If the model configuration doesn't specify any inputs or outputs
+  // then return error since pytorch backend does not support auto-completion.
+  size_t input_cnt = 0;
+  size_t output_cnt = 0;
+  {
+    triton::common::TritonJson::Value inputs;
+    if (ModelConfig().Find("input", &inputs)) {
+      input_cnt = inputs.ArraySize();
+    }
+
+    triton::common::TritonJson::Value outputs;
+    if (ModelConfig().Find("output", &outputs)) {
+      output_cnt = outputs.ArraySize();
+    }
+  }
+
+  if ((input_cnt == 0) || (output_cnt == 0)) {
+    return TRITONSERVER_ErrorNew(
+        TRITONSERVER_ERROR_INTERNAL,
+        "model configuration must contain inputs and outputs since "
+        "auto-complete is not supported for pytorch backend");
+  }
```

Comment:
The above should be remove as the validate functions are already responsible for checking if there are I/Os. Nothing should be done in autofill as PyTorch model doesn't provide model information.